dictionaries/powershell/package.json

```diff
@@ -11,8 +11,8 @@
     "./cspell-ext.json": "./cspell-ext.json"
   },
   "scripts": {
-    "build": "cspell-tools compile \"powershell.txt\" -o .",
-    "test": "head -n 100 \"powershell.txt\" | cspell -v -c ./cspell-ext.json \"--local=*\" \"--languageId=powershell\" stdin",
+    "build": "cspell-tools-cli compile --keep-raw-case src/powershell.txt --no-compress -o ./dict",
```

Comment:
I think `--use-legacy-splitter` is needed.

Reply:
Done!